tasks/pipeline.py

```diff
@@ -792,3 +792,74 @@ def trigger_build(ctx, branch_name=None, create_branch=False):
         print("Wait 10s to let Gitlab create the first events before triggering a new pipeline")
         time.sleep(10)
         run(ctx, here=True)
+
+
+@task(
+    help={
+        'owner-branch-name': 'Owner and branch names in the format <owner-name>/<branch-name>',
+        'no-verify': 'Adds --no-verify flag when git push',
+    }
+)
+def trigger_external(ctx, owner_branch_name: str, no_verify=False):
+    """
+    Trigger a pipeline from an external owner.
+    """
+    # Verify parameters
+    owner_branch_name = owner_branch_name.lower()
+
+    assert (
+        owner_branch_name.count('/') == 1
+    ), f'owner_branch_name should be "<owner-name>/<branch-name>" but is {owner_branch_name}'
+    assert "'" not in owner_branch_name
+
+    owner, branch = owner_branch_name.split('/')
+    no_verify_flag = ' --no-verify' if no_verify else ''
+
+    # Can checkout
+    status_res = ctx.run('git status --porcelain')
+    assert status_res.stdout.strip() == '', 'Cannot run this task if changes have not been committed'
+
+    # Get current branch
+    curr_branch_res = ctx.run('git branch --show-current', hide='stdout')
+    curr_branch = curr_branch_res.stdout.strip()
+
+    # Commands to push the branch
+    commands = [
+        # Fetch
+        f"git remote add '{owner}' 'git@github.com:{owner}/datadog-agent.git'",
```

Comment:
I'm not sure the `'` are required, if not it maybe improves the readability, can you double check? This remark is valid for below lines as well.
Thanks in advance
```suggestion
        f"git remote add {owner} git@github.com:{owner}/datadog-agent.git",
```